tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -105,6 +106,8 @@ private Security(SecuritySettings settings = null, InstrumentationGateway instru
                 {
                     Log.Information("AppSec remote enabling not allowed (DD_APPSEC_ENABLED=false).");
                 }
+
+                SharedRemoteConfiguration.AsmDDProduct.ConfigChanged += AsmDDProductConfigChanged;
```

Comment:
couldnt this one, on the otherhand,  be in the `EnabledWaf` only if security was enabled 🤔  ?